modules/ROOT/pages/dataweave-type-reuse-xml-schema.adoc

```diff
@@ -0,0 +1,430 @@
+
+The following example shows how to look for an XSD file in the path `org/weave/myfolder/User.xsd` inside your DataWeave script:
+
+----
+import * from xmlschema!org::weave::myfolder::User
+----
+
+== Import Syntax
+
+To import the XML schema types, as shown above, use the following syntax, where:
+
+* `typeToImport`: You can use `*` to import all types defined in the schema, or to import a single type from the schema, for example, `Root`. You can also import XML schema types with a different name, for example, `Root as Person`. This way  you can reference the type with that name in the script.
+* `pathToXsdSchemaFile`: To specify the path to the schema file, replace the file separators with `::` and remove the `.xsd` extension from the file name. For example, if the path to the schema is `example/schema/Person.xsd`, use `example::schema::Person`.
+
+----
+import _typesToImport_ from xmlschema!_pathToXsdSchemaFile_
+----
+
+The following example shows how to import a type:
+
+----
+import * from xmlschema!example::schema::Person
+----
+
+== Use Your Types in a DataWeave Script
+
+This example shows how to use types to import XML schema files into your DataWeave script.
+
+.XSD File (`User.xsd`):
+[source,XML]
+----
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema">
+    <xs:element name="Root">
+        <xs:complexType>
+            <xs:choice>
+                <xs:element name="employee" type="employee"/>
+                <xs:element name="member" type="member"/>
+            </xs:choice>
+            <xs:attribute name="id" use="required"/>
+        </xs:complexType>
+    </xs:element>
+    <xs:complexType name="employee">
+        <xs:sequence>
+            <xs:element name="name" type="xs:string"/>
+        </xs:sequence>
+    </xs:complexType>
+
+
+    <xs:complexType name="member">
+        <xs:sequence>
+            <xs:element name="id" type="xs:string"/>
+        </xs:sequence>
+    </xs:complexType>
+</xs:schema>
+----
+
+.DataWeave Script:
+[source,dataweave,linenums]
+----
+%dw 2.0
+import * from xmlschema!org::weave::myfolder::User
+
+
+{
+      Root @(id:"test") :{
+        employee: {
+            "name": "Mariano"
+        }
+      } match {
+             case is Root -> true
+             else -> false
+        }
+}
+----
+
+== From the XML Model to DataWeave Types
+
+As previously mentioned, the XML schema type system translates to DataWeave type directives. Therefore, as the two type models are considered different, the resulting DataWeave Type might not represent exactly the same XML Model.
```

Comment:
```suggestion
Because the XML schema type system translates to DataWeave type directives and the two type models are different, the resulting DataWeave type might not represent exactly the same XML model.
```

Reply:
Reworded.